django/api/services/calculate_rebate.py

```diff
@@ -0,0 +1,70 @@
+from ..models.go_electric_rebate_application import GoElectricRebateApplication
+from ..models.household_member import HouseholdMember
+from ..settings import INCOME_REBATES
+
+
+def calculate_rebate_amount(cra_response, application_id):
+    def check_individual(primary_income):
```

Comment:
does this need to be an inner function? I don't think it's taking any advantage of scoping from its parent. It could be its own unit tested function.

Reply:
do you want me to remove the check individual function and rewrite it where it gets called?

Reply:
moved